seqr/models.py

```diff
@@ -773,3 +782,38 @@ class Meta:
 
         json_fields = ['guid', 'name', 'description']
 
+
+class VariantSearch(ModelWithGUID):
+    name = models.CharField(max_length=200, null=True)
+    search = JSONField()
+
+    def __unicode__(self):
+        return self.name or self.id
+
+    def _compute_guid(self):
+        return 'VS%07d_%s' % (self.id, _slugify(self.name or ''))
+
+    class Meta:
+        unique_together = ('created_by', 'name')
+
+        json_fields = ['guid', 'name', 'search']
+
+
+class VariantSearchResults(ModelWithGUID):
+    variant_search = models.ForeignKey('VariantSearch', on_delete=models.CASCADE)
+    families = models.ManyToManyField('Family')
+    search_hash = models.CharField(max_length=50)
```

Comment:
would be nice to have docs about what info is included / not-included in search_hash and why it's needed instead of just using VariantSearchResults.guid